src/Test/Perf/util/TraceManager_util.csx

```diff
@@ -0,0 +1,231 @@
+#load "ScenarioGenerator_util.csx"
```

Comment:
`runner.csx` has a very specific job to just run the tests. `TraceManager` and related classes are just a helper in making those runs. So I believe `TraceManager` in a separate file will be good idea for both readability and keeping the file small.

Your thoughts?